examples/landmark_uncertainty/config.py

```diff
@@ -0,0 +1,114 @@
+"""
+Default configurations for uncertainty estimation using Quantile Binning
+"""
+
+from yacs.config import CfgNode as CN
+
+# -----------------------------------------------------------------------------
+# Config definition
+# -----------------------------------------------------------------------------
+
+_C = CN()
+
+# -----------------------------------------------------------------------------
+# Dataset
+# -----------------------------------------------------------------------------
+_C.DATASET = CN()
+_C.DATASET.SOURCE = (
+    "https://github.com/pykale/data/raw/main/tabular/cardiac_landmark_uncertainty/Uncertainty_tuples.zip"
+)
+# _C.DATASET.SOURCE = "https://github.com/pykale/data/blob/landmark-data/tabular/cardiac_landmark_uncertainty/Uncertainty_tuples.zip?raw=true"
+_C.DATASET.ROOT = "../../../data/landmarks/"
+_C.DATASET.BASE_DIR = "Uncertainty_tuples"
+
+
+_C.DATASET.FILE_FORMAT = "zip"
+
+_C.DATASET.CONFIDENCE_INVERT = [["S-MHA", True], ["E-MHA", True], ["E-CPV", False]]
+
+
+_C.DATASET.DATA = "4CH"
+_C.DATASET.LANDMARKS = [0, 1, 2]
+_C.DATASET.NUM_FOLDS = 8
+_C.DATASET.GROUND_TRUTH_TEST_ERRORS_AVAILABLE = True
+
+
+_C.DATASET.UE_PAIRS_VAL = "uncertainty_pairs_valid"
+_C.DATASET.UE_PAIRS_TEST = "uncertainty_pairs_test"
+
+
+# ---------------------------------------------------------------------------- #
+# Uncertainty Estimation Pipeline Parameters
+# ---------------------------------------------------------------------------- #
+_C.PIPELINE = CN()
+
+# Can choose to evalute over a single value or multiple values for Q (# bins). You can:
+# 1) Evaluate over each value of Q (set COMPARE_INDIVIDUAL_Q = True). For each Q it will compare DATASET.MODELS and DATASET.UNCERTAINTY_ERROR_PAIRS against eachother.
+# 2) Compare results of a single model and a single uncertainty error pair (set COMPARE_Q_VALUES = True).
+
+_C.PIPELINE.NUM_QUANTILE_BINS = [5, 10, 25]
+
+# ~# 1)
+# Compare uncertainty measures AND models over each single value of Q?
+_C.PIPELINE.COMPARE_INDIVIDUAL_Q = True
+# [NAME, KEY (error in CSV), KEY (uncertainty in csv)]
+_C.PIPELINE.INDIVIDUAL_Q_UNCERTAINTY_ERROR_PAIRS = [
+    ["S-MHA", "S-MHA Error", "S-MHA Uncertainty"],
+    ["E-MHA", "E-MHA Error", "E-MHA Uncertainty"],
+    ["E-CPV", "E-CPV Error", "E-CPV Uncertainty"],
+]
+# Key for model name found in path.
+_C.PIPELINE.INDIVIDUAL_Q_MODELS = ["U-NET", "PHD-NET"]
+# ~#
+
+
+# ~# 2)
+# Compare a single uncertainty measure on single model through various values of Q bins e.g. 5, 10, 20
+_C.PIPELINE.COMPARE_Q_VALUES = True
+_C.PIPELINE.COMPARE_Q_MODELS = ["PHD-NET"]  # Which model to compare over values of Q.
+_C.PIPELINE.COMPARE_Q_UNCERTAINTY_ERROR_PAIRS = [
+    ["E-MHA", "E-MHA Error", "E-MHA Uncertainty"]
+]  # Which uncertainty estimation measure to compare over values of Q.  # TODO: test to make sure this is 2D
```

Comment:
Is the TODO done? If not, are you going to do it? If not, put in TODO of the project board.